components/automate-cli/cmd/chef-automate/cmdUtils.go

```diff
@@ -132,6 +133,24 @@ func (c *remoteCmdExecutor) Execute() (map[string][]*CmdResult, error) {
 	return c.execute(c.NodeMap)
 }
 
+func filterNodes(nodeIps []string, filterNodes ...[]string) {
+	var filters []string
+	for _, f := range filterNodes {
+		filters = append(filters, f...)
+	}
+	var indexTobeRemoved []int
+	for i, nodeIp := range nodeIps {
+		for _, filter := range filters {
+			if strings.EqualFold(nodeIp, filter) {
+				indexTobeRemoved = append(indexTobeRemoved, i)
+			}
+		}
+	}
+	for _, idx := range indexTobeRemoved {
+		nodeIps = append(nodeIps[:idx], nodeIps[idx+1:]...)
```

Comment:
Think some simple logic for removal of particular index, Because in this line of code we reducing the size of slice (nodeIps) by one and we have indexes on the basis of older size of slice (nodeIps). I feel this will cause errors. Also write Unit test for the function